performant-software/textlab
# Adds Email Verification Support

**What this does**
Closes #6 
Adds email verification to signup process, supports re-verification

**How to test**
Go to: https://mel-textlab-pr-177.herokuapp.com/users/sign_in
1. Sign up for a new account
2. Confirm you recieve an email and can click-through to confirm
3. Test the "re-send confirmation" links

**IMPORTANT: (Developer-only) Notes for testing:**
1. If running heroku local without a local SSL, Links in emails will default to https- just change the link to http://, it will work properly on production and PR instances

2. You’re going to want to specify EMAIL_RETURN_PATH in your .env, probably this: EMAIL_RETURN_PATH=https://127.0.0.1:5000
 
Links for email confirmation include a token and if you hit a server other than what devise used to generate the token, it notices, so you can’t just change the server to test or confirm accounts
